HousingSearchApi/V1/Boundary/Requests/GetAllTenureListRequest.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace HousingSearchApi.V1.Boundary.Requests
+{
+    public class GetAllTenureListRequest : GetTenureListRequest
+    {
+        [FromQuery(Name = "lastHitId")]
+        public string LastHitId { get; set; }
+
+        //in order to use sorting together with last hit id, this must be provided as well 
```

Comment:
I've updated the validation in [#83d25a6](https://github.com/LBHackney-IT/housing-search-api/pull/226/commits/83d25a6a6d3ff630d84a212c80f20402ea4742f3), so that when `LastHitTenureStartDate` is provided then `LastHitId` must be provided as well. There's no dependency other way although the `lastHitId` in itself is not necessarily that useful without combining it with sorting in terms of getting predictable results. `LastHitId` can be used on its own however. I've also added validation for the `LastHitId` since it's the responsibility of this validator.